R/opts.R

```diff
@@ -320,10 +320,17 @@ trunc_opts <- function(dist = Fixed(0), default_cdf_cutoff = 0.001,
 #' conservative estimate of break point changes (alter this by setting
 #' `gp = NULL`).
 #'
-#' @param pop Integer, defaults to 0. Susceptible population initially present.
-#' Used to adjust Rt estimates when otherwise fixed based on the proportion of
-#' the population that is susceptible. When set to 0 no population adjustment
+#' @param pop A `<dist_spec>` giving the initial susceptible population size.
+#' Used to adjust Rt estimates based on the proportion of the population that
+#' is susceptible. Defaults to `Fixed(0)` which means no population adjustment
 #' is done.
+#' 
+#' @param pop_within_horizon Logical, defaults to `FALSE`. Should the
```

Comment:
This introduces an additional meaning for horizon, which elsewhere in the model is used only to describe the forecast horizon. Maybe `pop_within_data`?

Reply:
or maybe a more explicit `pop_period = c("forecast", "all")`? Or `pop_adjustment_period` or the like?